markdown/report.Rmd

```diff
@@ -166,121 +241,164 @@ More recently in December 2016 the ["Health and Social Care Delivery Plan"](http
 
-Over the nine years, 2010/11 to 2018/19 the percentage of time spent at home or in a community setting has increased from 85.3% to 89.2%, this equates to, on average an extra seven days within the last six months of life being spent at home or in a community setting in 2018/19 compared to 2010/11 (see Figure 1 below or [Appendix A1](#AppendixA1)).
+Over the `r as.character(as.english(year(end_date) - year(start_date)))` years, `r min(basefile$fy)` to `r latest$fy` the percentage of time spent at home or in a community setting has increased from `r basefile %>% summarise_data(include_years = "first") %>% pull(qom)`% to `r latest$qom`%, this equates to, on average an extra `r day_diff(basefile %>% summarise_data(include_years = "first") %>% pull(qom), latest$qom)` days within the last six months of life being spent at home or in a community setting in `r latest$fy` compared to `r min(basefile$fy)` (see Figure 1 below or [Appendix A1](#AppendixA1)).
 
-##### Figure 1: Percentage of last six months of life spent in hospital and at home or in a community setting; 2010/11 - 2018/19^1^
+##### Figure 1: Percentage of last six months of life spent in hospital and at home or in a community setting; `r min(basefile$fy)` - `r latest$fy`^1^
 
-##### Table here
+##### Figure 1 HERE
 
 <div custom-style = "Normal_Notes">
-1. Figures for 2018/19 are incomplete and will be revised in October 2019 publication.  See Appendix 2 for more information.
+1. Figures for `r latest$fy` are incomplete and will be revised in October `r year(pub_date)` publication.  See [Appendix 2](#Appendix2) for more information.
 </div>
 
+<br>
+
 ### Health Board of Residence
 Figure 2 shows the variation in the percentage of time spent at home or in the community in the last six months of life across Scotland by NHS Board. 
 
-The NHS Board where residents spent the highest percentage of time in their last six months of life at home or in the community was NHS Shetland where 94.2% of an individual’s last six months of life was spent at home or in a community setting and 5.8% in hospital. On average, this equates to approximately 11 days spent in a hospital setting within the last six months of life. 
+The NHS Board where residents spent the highest percentage of time in their last six months of life at home or in the community was `r max_hb$hb` where `r max_hb$qom`% of an individual’s last six months of life was spent at home or in a community setting and `r max_hb$qom_hosp`% in hospital. On average, this equates to approximately `r english(round_half_up(182.5 * (max_hb$qom_hosp / 100)))` days spent in a hospital setting within the last six months of life. 
```

Comment:
For `r english(round_half_up(182.5 * (max_hb$qom_hosp / 100)))` I notice it will still show a number (e.g. 11) even using english() as it's not a character. We need it to be a number anyway so can drop english()?

Reply:
Yes I think that is the rule - I've now written this into the day_diff function and dropped english from the above.